crates/http-source/src/websocket_source.rs

```diff
@@ -28,7 +28,7 @@ pub(crate) struct WebSocketSource {
 #[derive(Clone)]
 struct WSRequest {
     url: Url,
-    subscription_message: Option<String>,
+    subscription_messages: Option<Vec<String>>,
```

Comment:
Instead of carrying a Option and a Vec, it's probably cleaner to make this a plain Vec.  This makes the code later on just iterate over the vec. If the vec is empty, then the loop doesn't send anything.

Reply:
Updated to `Vec<String>` that defaults to `vec![]` if not provided.